regexes/bots.yml

```diff
@@ -1408,6 +1408,27 @@
 - regex: '(A6-Indexer|nuhk|TsolCrawler|Yammybot|Openbot|Gulper Web Bot|grub-client|Download Demon|SearchExpress|Microsoft URL Control|borg|altavista|teoma|blitzbot|oegp|furlbot|http%20client|polybot|htdig|mogimogi|larbin|scrubby|searchsight|seekbot|semanticdiscovery|snappy|vortex(?! Build)|zeal|fast-webcrawler|converacrawler|dataparksearch|findlinks|BrowserMob|HttpMonitor|ThumbShotsBot|URL2PNG|ZooShot|GomezA|Google SketchUp|Read%20Later|Minimo|RackspaceBot)'
   name: 'Generic Bot'
 
+- regex: 'Traackr.com'
+  name: 'Traackr.com'
+  category: 'Crawler'
+  url: 'http://www.traackr.com'
+
+- regex: 'feeltiptop'
+  name: 'feeltiptop.com'
+  category: 'Crawler'
+  url: ''
+  producer:
+    name: 'feeltiptop.com'
+    url: 'http://feeltiptop.com'
+
+- regex: 'WebClient/1.0'
+  name: 'WebClient'
+
+# Some libraries used for bots
+- regex: '(HTTP_Request2|Mechanize|aiohttp|Google-HTTP-Java-Client|WWW-Mechanize|faraday)'
+  name: 'Generic Bot'
```

Comment:
Guess this block moved in while merging. It shouldn't be present in that file anymore, so please remove it as well.